### PR TITLE
fix: linting errors fixed [#38]

### PR DESCRIPTION
**Pull request by anjaliahlawat**

### CODE REVIEW

**Comment by anjaliahlawat** [comment]
@h2grover Please check this out.
https://duncanleung.com/profile-measure-find-remove-slow-eslint-rules/
I checked it on my system and it seems that prettier is actually taking 82% of the total time spent. 
Maybe I should configure prettier like this. 
What do you think?

**Comment by h2grover** [comment]
@anjaliahlawat lets use eslint cache then. Please check that

**Comment by anjaliahlawat** [comment]
lint-errors
@h2grover In my system, on linting,it is showing around 4800 errors. Is your system not showing these errors?

**Comment by anjaliahlawat** [comment]
@h2grover I added .eslintignore file and added some directories which I thought don't require linting. Now  I am not getting any errors. Please review it and let me know if it's the right way to go.

File: .eslintignore [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Gauge - log files dir
+logs
+
```
Comment (h2grover) [inline .eslintignore:3]: Please add node_modules also

**Review by h2grover — CHANGES_REQUESTED** [review]

File: .eslintignore (hunk 1 above)
Comment (anjaliahlawat) [reply .eslintignore:3]: In eslint documentation, it is given that by default node_modules and any files or directories starting with . are ignored while linting.

Comment (h2grover) [reply .eslintignore:3]: Ok thanks for cross checking @anjaliahlawat.

Thats' weird I can see errors on my machines, I will cross check it tomorrow then

**Review by h2grover — APPROVED** [review]
LGTM

**Pull request merged**